src/scope/Operator.js

```diff
@@ -16,6 +16,11 @@ import User from '../entity/User'
 import Batch from '../entity/Batch'
 import Place from '../entity/Place'
 import File from '../entity/File'
+import OpeatorAccess from '../entity/OperatorAccess'
```

Comment:
`OperatorAccess` is a sub-resource of `Account`, so not needed here

Reply:
fixed, thanks!